tests/test_group.py

```diff
@@ -74,8 +74,7 @@ def test_group_type_change(self, base_url, selenium, vouched_user):
         assert group_type.is_member_criteria_visible
 
     @pytest.mark.credentials
-    @pytest.mark.xfail("'allizom.org' in config.getvalue('base_url')",
-                       reason='https://github.com/mozilla/mozillians-tests/issues/231')
+    @pytest.mark.skip(reason='https://github.com/mozilla/mozillians-tests/issues/231')
```

Comment:
No need to change this to a skip, just set `run=False` as [documented here](http://pytest.org/latest/skipping.html#run-parameter).